src/components/layout/homepage/mobile-links/Styles.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+import { Link as Link_ } from "react-router-dom";
+import { colors } from "../../../utils/common/CommonStyles";
+
+export const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 125px;
+
+  @media screen and (min-width: 450px) {
+    display: none;
+  }
+`;
+
+export const LinksContainer = styled.div`
+  display: flex;
+`;
+
+export const Link = styled(Link_)`
+  margin: 0 5px;
+  text-decoration: none;
+  color: ${colors.black};
+  padding: 10px;
+  background-color: ${colors.sand};
```

Comment:
Great job exporting your colors from a global color file, but try naming them something a bit more descriptive than sand. Usually you'll have a primary color palette and you'll name them "primary-light" "primary-dark" "primary-orange" "primary-dark-orange" and so on.